src/app/task/taskPriorityType.ts

```diff
@@ -1,8 +1,36 @@
-export enum PriorityEnum {
-  Urgent = 'invert(9%) sepia(87%) saturate(7293%) hue-rotate(360deg) brightness(93%) contrast(126%)',
-  Heigh = 'invert(49%) sepia(84%) saturate(1728%) hue-rotate(358deg) brightness(100%) contrast(108%)',
-  Medium = 'invert(49%) sepia(72%) saturate(1226%) hue-rotate(68deg) brightness(97%) contrast(99%)',
-  Low = 'invert(75%) sepia(91%) saturate(729%) hue-rotate(138deg) brightness(104%) contrast(104%)',
+import { TaskPriorityInterface } from "./taskPriority.interface";
+
+export enum TaskPriorityTitleEnum {
+  Urgent = "Urgent",
+  Height = "Height",
+  Medium = "Medium",
+  Low = "Low",
+};
+
+export enum TaskPriorityColorEnum {
+  Urgent = "invert(9%) sepia(87%) saturate(7293%) hue-rotate(360deg) brightness(93%) contrast(126%)",
+  Height = "invert(49%) sepia(84%) saturate(1728%) hue-rotate(358deg) brightness(100%) contrast(108%)",
+  Medium = "invert(49%) sepia(72%) saturate(1226%) hue-rotate(68deg) brightness(97%) contrast(99%)",
+  Low = "invert(75%) sepia(91%) saturate(729%) hue-rotate(138deg) brightness(104%) contrast(104%)",
 }
 
-export type TaskPriorityType = PriorityEnum | string;
\ No newline at end of file
+export type TaskPriorityTitleType = keyof typeof TaskPriorityTitleEnum;
+
+export const TaskPriorityData: Record<TaskPriorityTitleEnum, TaskPriorityInterface> = {
```

Comment:
we have discussed that it could be an array of task priorities  TaskPriorityInterface[] without enums, just consts.